docs/source/plug-in.md

```diff
@@ -0,0 +1,26 @@
+## Other modes
+
+The `KernelGatewayApp.api` can be set to the name of any public module supplying a personality. This allows for suporting alternate kernel communications mechanisms.
+
+### Factory function
+`create_personality(*args, **kwargs)`
```

Comment:
You should name the `parent` argument here, even though the implementation does **kwargs as a passthrough.